includes/shortcodes/class.llms.shortcodes.php

```diff
@@ -5,7 +5,8 @@
  * @package LifterLMS/Classes/Shortcodes
  *
  * @since 1.0.0
- * @version 6.0.0
+ * @since [version] Added self instantiation.
```

Comment:
```suggestion
```

Coding standards don't allow for a changelog in the fileheader (except for template files).

In this scenario we're okay to not document this change. It's okay.

Reply:
Please don't shoot the messenger.

The file header example changelog in the LifterLMS documentation standards has multiple `@since` tags with one that says "Description of file changes".
https://github.com/gocodebox/lifterlms/blob/trunk/docs/documentation-standards.md#file-headers-1

The commit is changing to the file's behavior.

Reply:
https://github.com/gocodebox/lifterlms/blob/trunk/docs/documentation-standards.md#file-headers

And run PHPCS (which correctly reports the error. phpcs is not wrong in this case)

File header changelogs don't keep a changelog because it's *generally* redundant... You've found an exception to this rule and I've asked you to ignore it.

The example is probably a hold over from when we *did* (stupidly) keep redundant changelogs. *BUT* we do keep a changelog on file headers for certain files (templates and views, for example).

So.... apply my suggestion, please.

Reply:
@thomasplevy, I think I understand what happened. When I run PHPCS, I usually use `composer check-cs-errors` instead of `composer check-cs`. With `check-cs-errors`, the `--error-severity=1 --warning-severity=6` options cause PHPCS to not report multiple `@since` tags in the file header as an error.

I probably got it in my head from the [pull request template](https://github.com/gocodebox/lifterlms/blob/trunk/.github/PULL_REQUEST_TEMPLATE.md?plain=1) that `check-cs-errors` should be used. I'll use `check-cs` from now on.

Reply:
No worries @pondermatic!